client/init_test.go

```diff
@@ -24,5 +24,5 @@ import (
 // This file initializes the blockchain and logging backend for both, whitebox
 // and blackbox tests (files *_test.go in packages client and client_test).
 func init() {
-	plogrus.Set(logrus.WarnLevel, &logrus.TextFormatter{ForceColors: true})
+	plogrus.Set(logrus.TraceLevel, &logrus.TextFormatter{ForceColors: true})
```

Comment:
Did you intended to change the logging level permanently or is it only for testing and you missed to revert ?

Reply:
Intended, will place it in a separate commit.